javacutil/src/main/java/org/checkerframework/javacutil/AnnotationBuilder.java

```diff
@@ -286,7 +287,14 @@ public AnnotationMirror build() {
     public void copyElementValuesFromAnnotation(
             AnnotationMirror valueHolder, String... ignorableElements) {
         Set<String> ignorableElementsSet = new HashSet<>(Arrays.asList(ignorableElements));
-        copyElementValuesFromAnnotation(valueHolder, ignorableElementsSet);
```

Comment:
This method was changed.  I'm curious why the call didn't remain but was replaced by a `for` loop.